app/ports/templates/ports/maintainerdetail.html

```diff
@@ -58,23 +58,36 @@ <h3 id="maintainers__name">{{ maintainer }}</h3>
            placeholder="Search Ports by '{{ maintainer }}'">
     <br>
     <div id="all_ports_table">
-        <h3>Total {{ all_ports_num }} Ports Maintained By : <em>{{ maintainer }}</em></h3>
-        <form class="form-inline"></form>
+        <h3>Ports maintained By : <em>{{ maintainer }}</em></h3>
+        <div class="row">
+                <div class="col-lg-7">
+                    <p>Page {{ ports.number }} of {{ ports.paginator.num_pages }} | Showing
+                        ports {{ ports.start_index }} to
+                        {{ ports.end_index }} of {{ all_ports_num }}</p>
+                </div>
+                <div class="col-lg-5">
+                    <form class="form-inline justify-content-lg-end" method="get"
+                            action="">
+                        <label for="paginate_by">Ports per page: </label>
+                        <input style="max-width: 150px" class="ml-2" type="number" name="paginate_by"
+                               value="{{ paginate_by }}">
+                        <button type="submit">Update</button>
+                    </form>
+                </div>
+        </div>
```

Comment:
Okay, I will use `includes` for this and some other places. But we won't be able to use it everywhere, because for example at [/statistics/ports/](https://ports.macports.org/statistics/ports/) pagination makes an ajax call and the template we use everywhere else will not be usable here.